attractors/attractors.ipynb

```diff
@@ -243,7 +256,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "@jit(nopython=True)\n",
+    "@jit()\n",
```

Comment:
I think `@jit` is sufficient, and is preferable to `@jit()` (here and elsewhere).